build.gradle

```diff
@@ -71,18 +70,16 @@ task configureGroovyAll {
             'MANIFEST_MF'                   : 'META-INF/MANIFEST.MF'
         ]
 
-        def collectDuplicates = { File[] roots ->
+        def findDuplicates = { List<FileTree> fileForest ->
```

Comment:
It's a bit weird that these methods are defined like this. Can we move them to be (maybe `static`) methods at the top level?

Reply:
I can't create static methods in a .gradle file

Reply:
Why not?

Reply:
Because they depend on static state and can't create constants in the root of the build file:
<img width="310" alt="Screen Shot 2019-07-24 at 13 58 23" src="https://user-images.githubusercontent.com/1841944/61791753-2c2ce300-ae1b-11e9-90ac-1ca8c5036c05.png">

Reply:
I'll merge it to be able to test the artifact, I'll address further requests in a separate PR

Reply:
That's a static field, and indeed that won't work. But static methods do.

Reply:
In general they do, my statement was ambiguous, I meant I can't push these methods, since they depend on static state that I can't use otherwise